coreneuron/permute/cellorder.cpp

```diff
@@ -583,65 +576,65 @@ int temp3[1024] = {0};
  * warp deals with a group of cells, therefore multiple compartments (finer level of parallelism).
  */
 void solve_interleaved2(int ith) {
-    static int foo = 1;
     NrnThread* nt = nrn_threads + ith;
     InterleaveInfo& ii = interleave_info[ith];
     int nwarp = ii.nwarp;
-    if (nwarp == 0) {
+    if (nwarp == 0)
         return;
-    }
-    int* ncycles = ii.cellsize;         // nwarp of these
-    int* stridedispl = ii.stridedispl;  // nwarp+1 of these
-    int* strides = ii.stride;           // sum ncycles of these (bad since ncompart/warpsize)
-    int* rootbegin = ii.firstnode;      // nwarp+1 of these
-    int* nodebegin = ii.lastnode;       // nwarp+1 of these
-#ifdef _OPENACC
-    int nstride = stridedispl[nwarp];
-    int stream_id = nt->stream_id;
-#endif
 
     int ncore = nwarp * warpsize;
+
 #ifdef _OPENACC
-    // clang-format off
+    if (corenrn_param.gpu && corenrn_param.cuda_interface) {
+        auto* d_nt = static_cast<NrnThread*>(acc_deviceptr(nt));
+        auto* d_info = static_cast<InterleaveInfo*>(acc_deviceptr(interleave_info + ith));
```

Comment:
I haven't checked locally [yet], but I suspect the errors in the unified memory Jenkins build might be because `nullptr` is returned here.